python/fusion_engine_client/utils/argument_parser.py

```diff
@@ -151,7 +152,12 @@ def _format(option):
                 elif '=' in option:
                     return None
                 else:
-                    return f"{option}[=<{', '.join(TriStateBooleanAction.POSSIBLE_VALUES)}>]"
+                    nonlocal options_printed
```

Comment:
Huh I've never seen this `nonlocal` keyword before - pretty neat.

Reply:
Yeah it's not a super common thing, and they added it in Python 3. It lets you modify a primitive variable that lives outside the scope of a nested function. Before that in Python 2 you had to do ugly stuff to make your primitive into an object to work around the limitation. Something like:
```py
my_number = [3]
def my_func():
    my_number[0] += 5
```